frontend/src/pages/spotnet/dashboard/Dashboard.jsx

```diff
@@ -192,7 +88,7 @@ const Dashboard = ({ walletId }) => {
         ))}
       </div>
       <div>
-        <button className="btn redeem-btn border-0" onClick={() => closePositionEvent()}>
+        <button className="btn redeem-btn border-0" onClick={() => closePositionMutation.mutate()}>
```

Comment:
you should disable button on `closePositionMutation.isPending`, so it can't be clicked multiple times with the same mutation

Reply:
Access to the button is fixed. Please check it out. Thank you.